doc/release-notes-1.1.0.rst

```diff
@@ -0,0 +1,621 @@
+    * If a debugger (for example, J-Link) is connected via SWD to the nRF9160, the modem firmware will reset.
+      Therefore, the LTE modem cannot be operational during debug sessions.
+    * The SEGGER Control Block cannot be found by automatic search by the RTT Viewer/Logger.
+      As a workaround, set the RTT Control Block address to 0 and it will try to search from address 0 and upwards.
+      If this does not work, look in the ``builddir/zephyr/zephyr.map`` file to find the address of the ``_SEGGER_RTT`` symbol in the map file and use that as input to the viewer/logger.
+    * nRF91 fails to receive large packets (over 4000 bytes) over NB-IoT. LTE-M is not affected.
+    * nrf_connect fails if called immediately after initialization of the device. A delay of 1000 ms is required for this to work as intended.
+
+Crypto
+======
+
+.. note::
+   Old - still valid?
+
+   * The :ref:`nrfxlib:nrf_security` glue layer is broken because symbol renaming is not handled correctly.
+     Therefore, the behavior is undefined when selecting multiple back-ends for the same algorithm (for example, AES).
+
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+* :option:`CONFIG_BT_HCI_TX_STACK_SIZE` must be set to 1536 when selecting :option:`CONFIG_BT_LL_NRFXLIB`.
+* The :ref:`nrfxlib:ble_controller` 0.3.0-3.prealpha might assert when receiving a packet with an CRC error on LE Coded PHY after performing a DLE procedure where RX Octets is changed to a value above 140.
+* :option:`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE` must be set to 2048 when selecting :option:`CONFIG_BT_LL_NRFXLIB` on :ref:`central_uart` and :ref:`central_bas`.
+* :option:`CONFIG_NFCT_IRQ_PRIORITY` must be set to 5 or less when selecting :option:`CONFIG_BT_LL_NRFXLIB` on :ref:`peripheral_hids_keyboard`.
+* When selecting :option:`CONFIG_BT_LL_NRFXLIB`:
+  If a directed high duty cycle advertiser times out, the application might have to wait a short time before starting a new connectable advertiser.
+  Otherwise, starting the advertiser will fail.
+* Bluetooth Low Energy peripheral samples are unstable in some conditions (when pairing and bonding are performed, and when disconnections/re-connections happen).
+
+.. note::
+   Old - which of these are still valid?
+
+    * :ref:`peripheral_lbs` does not report the Button 1 state correctly.
+    * The central samples (:ref:`central_uart`, :ref:`bluetooth_central_hids`) do not support any pairing methods with MITM protection.
+    * On some operating systems, the nrf_desktop application is unable to reconnect to a host.
+    * central_uart: A too long 212-byte string cannot be handled when entered to the console to send to peripheral_uart.
+    * central_hids: After flashing a HEX file to the nrf52_pca10040 board, UART connectivity is lost when using the BLE Controller. The board must be reset to get UART output.
```

Comment:
This works for me, so I think we should remove it. @qung can you still see this issue?

Reply:
Yes, we should remove these lines